src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromPartner.tsx

```diff
@@ -22,35 +24,24 @@ export class ArtworkDetailsAboutTheWorkFromPartner extends React.Component<
     if (!additional_information) {
       return null
     }
-    const locationNames =
-      partner &&
-      partner.locations &&
-      partner.locations.length > 0 &&
-      filterLocations(partner.locations)
+
+    const locationNames = get(
+      partner,
+      p => filterLocations(p.locations),
+      []
+    ).join(", ")
+
+    const imageUrl = get(partner, p => p.profile.image.url, false)
```

Comment:
This looks readable 👌

I am wondering about that fallback value, seems a bit weird to have a string or false.

Reply:
Updating `valueIfFailOrUndefined?: T` (from `any` to `T`) actually caught this bug nicely; updated. (Also good to note that we don't need a fallback value in that third spot.)